### PR TITLE
Bump DS alpha.78 (no breaking changes)

### PR DESCRIPTION
**Pull request by HichamELBSI**
- Rename `size` to `spacing` for Stack component (to remove the deprecation warning in the console)
- Rename `toggle` to `onToggle` for Stack component (to remove the deprecation warning in the console)
- Bump DS version `0.0.1-alpha.78`

### CODE REVIEW

File: packages/core/admin/admin/src/StrapiApp.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ class StrapiApp {
       locales: ['en'],
       menuLogo: MenuLogo,
       notifications: { releases: true },
-      theme: lightTheme,
+      theme: darkTheme,
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/StrapiApp.js:37]: I guess it was for a test, shouldn't we remove it for now? 😅

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
Thaaanks
Small leftover from dark mode testing I think 😊

**Review by soupette — DISMISSED** [review]

**Review by ronronscelestes — DISMISSED** [review]
LGTM!

**Review by gu-stav — DISMISSED** [review]
Looks good to me. Thank you 🌻

**Review by ronronscelestes — APPROVED** [review]
LGTM

**Pull request merged**